Jenkinsfile

```diff
@@ -58,10 +57,15 @@ Or **/*x64_linux*.tar.gz,**/*x64_linux*.sha256.txt,**/*x64_linux*.json,**/*x64_l
                             excludes: "${params.ARTIFACTS_TO_SKIP}",
                             projectName: "${upstreamJobName}",
                             selector: [$class: 'SpecificBuildSelector', buildNumber: "${upstreamJobNumber}"]])
-                        sh '''
-                        export VERSION=`echo $VERSION | awk '{print toupper($0)}'`
-                        JAVA_HOME=/usr/lib/jvm/java-8-openjdk-amd64 ./sbin/Release.sh
-                        '''
+                        
+                        withCredentials([string(credentialsId: "${params.GITHUB_CREDENTIAL}", variable: 'secretToken')]) {
+                            withEnv(["GITHUB_TOKEN=${secretToken}"]) {
+                                sh '''
+                                    export VERSION=`echo $VERSION | awk '{print toupper($0)}'`
+                                    JAVA_HOME=/usr/lib/jvm/java-8-openjdk-amd64 ./sbin/Release.sh
```

Comment:
Can we rely on a more modern Java here?

Reply:
This is the java the current job is using  https://ci.adoptium.net/job/build-scripts/job/release/job/refactor_openjdk_release_tool/. Which modern java is available and can be used in https://ci.adoptium.net/computer/C3jenkins/? @sxa